rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleServiceNotFoundException.java

```diff
@@ -7,15 +7,11 @@
     private final UUID serviceUUID;
 
     public BleServiceNotFoundException(UUID serviceUUID) {
+        super("BLE Service not found with uuid " + serviceUUID);
```

Comment:
I think that `uuid` should be definitely with uppercase (I mean: `UUID` instead of `uuid`), as it is more frequent used version then with lowercase, you used is in another class  and  it relates to used there class name.

Reply:
Correct, I'll update in another commit.